kpi/tasks.py

```diff
@@ -95,15 +95,13 @@ def enketo_flush_cached_preview(server_url, form_id):
     )
     response.raise_for_status()
 
-
-@celery_app.task
-def remove_asset_snapshots(asset_id: int):
+@celery_app.task(time_limit=82800, soft_time_limit=82800)
+def perform_maintenance():
     """
-    Temporary task to delete old snapshots.
-    TODO remove when kpi#2434 is merged
+    Run daily maintenance tasks
     """
-    call_command(
-        'delete_assets_snapshots',
-        days=constance.config.ASSET_SNAPSHOT_DAYS_RETENTION,
-        asset_id=asset_id,
-    )
+    from kobo.apps.markdownx_uploader.tasks import remove_unused_markdown_files
```

Comment:
Circular import? Might be good to note. Did you spend any time looking into why?